functional/kata-monitor/run.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/sh
+#
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test file will test kata-monitor for basic functionality (retrieve kata sandboxes)
+# It will assume an environment where:
+# - a CRI container manager (container engine) will be up and running
+# - crictl is installed and configured
+# - the kata-monitor binary is available on the host
+#
+
+CRI_RUNTIME=${CRI_RUNTIME:-"crio"}
+RUNTIME=${RUNTIME:-"containerd-shim-kata-v2"}
+KATA_MONITOR_BIN=${KATA_MONITOR_BIN:-$(type -p kata-monitor)}
+IAM=$(whoami)
+TMPATH=$(mktemp -d -t kata-monitor-test-XXXXXXXXX)
+METRICS_FILE="${TMPATH}/metrics.txt"
+MONITOR_LOG_FILE="${TMPATH}/kata-monitor.log"
+CACHE_UPD_TIMEOUT_SEC=${CACHE_UPD_TIMEOUT_SEC:-10}
+PODID=""
+CONID=""
+
+FALSE=1
+TRUE=0
+
+title() {
+	local STEP="$1"
+	echo "* STEP: $STEP"
+}
+
+echo_ok() {
+	local MSG="$1"
+
+	echo "OK: $MSG"
+	echo ""
+}
+
+error_with_msg() {
+	local MSG=${1}
+
+	echo -e "\nERROR: $MSG"
+	echo -e "\nkata-monitor logs:\n----------------"
+	cat "$MONITOR_LOG_FILE"
+	cleanup
+	echo -e "\nkata-monitor testing: FAILED!"
+	exit 1
+}
+
+cleanup() {
+	crictl stop "$CONID" "$VANILLA_CONID"  > /dev/null 2>&1
+	crictl rm "$CONID" "$VANILLA_CONID"    > /dev/null 2>&1
```

Comment:
Nit: It's rather distracting having the redirects on every call. Maybe you could use `chronic` or create a `run_command()` function that handles the redirect?

Reply:
I redirect /dev/null only for crictl... so created a `qcrictl()` function (quiet crictl)!